app/mailers/notifier_mailer.rb

```diff
@@ -37,18 +37,22 @@ def all_details
      @owner_mail=Owner.find(@owner).email
     end
 
+    private
     def menu_items
        @cost = 0 
        order_all_details = ""
        @menu.split("+") do |o| 
          str= o.split("*")
          @cost = @cost + str[2].to_i
-         order_all_details = order_all_details + "#{str[0]}  X #{str[1]} = #{str[2]} ₹\n"
+         order_all_details = order_all_details + "#{str[0]} "+ AllConstants::QUANTITY + " #{str[1]} " + AllConstants::ASSIGN + " #{str[2]} " + AllConstants::COST + "\n"
```

Comment:
#{} this is already used for cancat operation why separate + operation is done?

Reply:
because if i don't use + it will take AllConstants::QUANTITY as string and print the same but actually AllConstants::QUANTITY refers to symbol "X"

Reply:
changed